Overview.src.html

```diff
@@ -4178,44 +4233,92 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
+  <p>To <span>process response</span> for <var>response</var>, run these substeps:
 
   <ol>
-   <li><p>If <var title>response</var>'s <span title=concept-response-type>type</span> is
-   "<code title>error</code>", reject <var title>p</var> with a
-   <code title>TypeError</code>.
+   <li><p>If <var>response</var>'s <span title=concept-response-type>type</span> is
+   "<code title>error</code>", reject <var>p</var> with a <code title>TypeError</code> and abort
+   these steps.
 
-   <li><p>Otherwise, resolve <var title>p</var> with a new <code>Response</code> object
-   associated with <var title>response</var> and a new <code>Headers</code> object whose
+   <li><p>Let <var>res</var> be a new <code>Response</code> object associated with
+   <var>response</var> and a new <code>Headers</code> object whose
    <span title=concept-Headers-guard>guard</span> is "<code title>immutable</code>".
+
+   <li><p>If <var>response</var>'s status is a <span>null body status</span>, resolve <var>p</var>
+   with <var>res</var> and abort these steps.
+
+   <li><p>Let <var>pull</var> be an action that <span title=concept-fetch-resume>resumes</span> the
+   ongoing fetch if it is <span title=concept-fetch-suspend>suspended</span>.
+
+   <li><p>Let <var>cancel</var> be an action that
+   <span title=concept-fetch-terminate>terminates</span> the ongoing fetch with reason
+   <i title>end-user abort</i>.
+
+   <li><p>Let <var>strategy</var> be an object. The user agent may choose any object.
+   <p class="note no-backref"><var>strategy</var> is used to control the queuing strategy of
+   <var>stream</var> constructed below.
+
+   <li><p>Set <var>stream</var> to the result of
+   <span title=concept-construct-ReadableStream>constructing</span> a
+   <span title=concept-ReadableStream>ReadableStream</span> with <var>strategy</var>,
+   <var>pull</var> and <var>cancel</var>. If that threw an exception, run the following
+   substeps:
+   <ol>
+    <li><p>Reject <var>p</var> with that exception.
+
+    <li><p><span title=concept-fetch-terminate>Terminate</span> the ongoing fetch with reason
+    <i title>fatal</i>.
+   </ol>
+
+   <li><p>Otherwise, run the following substeps:
+   <ol>
+    <li><p>Set <var>res</var>'s <span title=concept-Response-readable-stream>readable stream</span>
+    to <var>stream</var>.
+
+    <li><p>Resolve <var>p</var> with <var>res</var>.
+   </ol>
   </ol>
+  <p>To <span>process response body</span> for <var>response</var>, run these substeps:
+  <ol>
+   <li><p>If <var>stream</var> is null, abort these steps.
+
+   <li><p><span title=concept-enqueue-ReadableStream>Enqueue</span> a <code>Uint8Array</code> whose
+   contents are <var>response</var>'s <span title=concept-response-body>body</span> into
+   <var>stream</var>.
 
-  <p>To <span>process response body</span> for <var title>response</var>, do nothing.
+   <li><p>Clear out <var>response</var>'s <span title=concept-response-body>body</span>.
```

Comment:
I thought it might be unclear if reading is an idempotent operation or a destructive operation. If it's clear that reading from a byte stream is destructive, we can remove the sentence. Done.